### PR TITLE
completed css-flexbox

### PR DESCRIPTION
**Pull request by Quintin-Russell**
snippit tool broken, it's good i promise

### CODE REVIEW

File: css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    #card-deck {
+      display: flex;
+      flex-basis: calc(100% / 3);
```
Comment (BrettAlbright) [inline css-flexbox/align-items.html]: This flex-basis needs to be put on the child elements, the `card-container` class. Flex-basis is a property which sets the width of flex items.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Small update needed!

File: css-flexbox/display-flex.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - display: flex</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    #card-deck {
+      display: flex;
+      padding: 1em;
+    }
+
```
Comment (BrettAlbright) [inline css-flexbox/display-flex.html:15]: missing the padding on the child elements

File: css-flexbox/flex-wrap.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100% / 6);
```
Comment (BrettAlbright) [inline css-flexbox/flex-wrap.html]: This needs to be on the child elements

File: css-flexbox/justify-content.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    #card-deck {
+      display: flex;
+      flex-basis: 25%;
```
Comment (BrettAlbright) [inline css-flexbox/justify-content.html]: must be on the child elements

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**